docs/index.md

```diff
@@ -65,48 +60,45 @@ var vlSpec = {
       }
     };
 
-var vgSpec = vl.compile(vlSpec).spec;
-vg.parse.spec(vgSpec, function(chart) {
-  chart({el:"#vis"}).update(); });
+  // Use Vega-Lite to compile the Vega-Lite specification to Vega
+  var vgSpec = vl.compile(vlSpec).spec;
 
-</script>
+  // Use Vega to display the visualization
+  vg.parse.spec(vgSpec, function(chart) {
+    chart({el:"#vis"}).update(); });
+  </script>
+</body>
+</html>
 ```
 
+Basically, Vega-Lite compiles a Vega-Lite specification into a Vega
+specification and use Vega Runtime to display visualizations. You can see the result [here](demo.html).
 
-### Data from URL
-
-Here is the bare minimum html file to get Vega-Lite with data from url working in a webpage.
-
-```html
-<!DOCTYPE html>
-<title>Vega Lite Bar Chart</title>
-<meta charset="utf-8">
+The page first loads the required libraries (D3, Vega and Vega-Lite).
+Then we created an HTML element that we will render the visualization in.
+In JavaScript, the Vega-Lite specification is compiled to Vega.
+Lastly, the Vega specification is passed to the [Vega runtime](https://github.com/vega/vega/wiki/Runtime), which renders it into the `#vis` element.
 
-<script src="../lib/d3.min.js"></script>
-<script src="../lib/vega.js"></script>
-<script src="../lib/vega-lite.js"></script>
+If viewed in a browser, this will display a simple bar chart.
+This example shows only the basic set up. The rest of the documentation describes the Vega-Lite specification language in detail.
```

Comment:
> The rest of the documentation describes the Vega-Lite specification language in detail.

This makes me wonder if this page should be just an index of links like Vega wiki's index page: https://github.com/vega/vega/wiki